sdk/core/azure-core/inc/azure.hpp

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+/**
+ * @brief Add main headers from Azure.
+ *
+ *
+ */
+
+#include "azure/response.hpp"
```

Comment:
Are we sure we want this? I think we should leave this out, especially since it wouldn't contain all of azure.

Reply:
Agreed in the namespaces meeting

Reply:
I agree, this might be confusing - you include azure.hpp, and it is not the whole azure.
Better keep them added to core.hpp, IMO.

Reply:
> Agreed in the namespaces meeting

Maybe the recording ended early, but I didn't hear about this change in the meeting. Let's revert such changes form this PR and revisit separately if you feel strongly.